experimental/auth-logic/provenance_build_wrapper.go

```diff
@@ -25,40 +25,49 @@ import (
 
 type provenanceBuildWrapper struct{ provenanceFilePath string }
 
-func (pbw provenanceBuildWrapper) EmitStatement() UnattributedStatement {
-	handleErr := func(err error) {
-		if err != nil {
-			panic(err)
-		}
-	}
-
+func (pbw provenanceBuildWrapper) EmitStatement() (UnattributedStatement, error) {
 	// Unmarshal a provenance struct from the JSON file.
-	provenance, provenanceParseErr := slsa.ParseProvenanceFile(pbw.provenanceFilePath)
-	handleErr(provenanceParseErr)
+	provenance, err := slsa.ParseProvenanceFile(pbw.provenanceFilePath)
+	if err != nil {
+		return UnattributedStatement{},
+			fmt.Errorf("provenance build wrapper couldn't parse provenance file:%s", err)
```

Comment:
```suggestion
			fmt.Errorf("provenance build wrapper couldn't parse provenance file: %v", err)
```

(as per https://g3doc.corp.google.com/go/g3doc/style/best-practices.md?cl=head#error-extra-info )

Reply:
fixed